src/plugins/npm/npmPlugin.test.js

```diff
@@ -0,0 +1,31 @@
+const npmPlugin = require('./npmPlugin');
+
+
+async function testNpm(message) {
+  return new Promise((resolve) => {
+    npmPlugin({
+      command: { command: message },
+      respondWithMention: resolve,
+      handling() {}
+    });
+  });
+}
+
+it('works', async () => {
+  const output = await testNpm('npm bootstrap');
+
+  expect(output.split('|')[0]).toEqual('npm.im/bootstrap');
+  expect(output.split('|')[1]).toMatch(/^\d+\.\d+\.\d+/);
+  expect(output.split('|')[2]).toMatch(/^\d{4}-\d{2}-\d{2}$/);
+});
+
+it('works for searches', async () => {
+  const output = await testNpm('npm ~bootstrap');
+
+  const results = output.split('\n');
+  expect(results.length).toBeGreaterThan(2);
+
+  expect(results[0].split('|')[0]).toEqual('npm.im/bootstrap');
+  expect(results[0].split('|')[1]).toMatch(/^\d+\.\d+\.\d+/);
+  expect(results[0].split('|')[2]).toMatch(/^\d{4}-\d{2}-\d{2}$/);
+});
```

Comment:
please ensure all files have trailing newlines

Reply:
Done, but we could maybe have done those things using eslint rule eol-last, no-trailing-spaces, comma and without the terrible prettier plugin

Reply:
i totally agree, and would be fine to remove prettier later - it was just using the prettier config already which disabled those rules, so i went with that.